src/components/RecentSearches/RecentSearches.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { RecentSearchesContainer, RdecentHeader, ClearButton, RdecentBody, SearchItem, DeleteItemButton } from "./RecentSearches-style";
+import ExitIcon from '../../assets/Icons/exit.svg';
+
+const RecentSearches = () => {
+
+    const recentSearch = ['amir', 'yehuda', 'ofek'];
+    const click = () => {
+        console.log('amir')
+    }
+
+    const showLastSearches = () => {
+        return recentSearch.map((searchTitle, index) => (
+            <>
+                <SearchItem>
+                    {searchTitle}
+                    <DeleteItemButton src={ExitIcon} onClick={click}/>
```

Comment:
Generally, the functions that pass on click event calls '<action>handler'. so here you can call the function 'onDeleteHandler'.

Give everything (function or variables) as informative names as possible, when you start with functionality implementation it will help you, and others understand the flow.